activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+*   Add expiry and purpose to MessageEncryptor & MessageVerifier.
```

Comment:
Let's say `purpose and expiry support` and wrap the verifier and encryptor in backticks.